src/createServer.js

```diff
@@ -1,3 +1,38 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { getErrors } = require('./getErrors');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const currentURL = new URL('htto://' + req.url);
+    const reqText = currentURL.pathname.slice(1);
+    const reqCase = currentURL.searchParams.get('toCase');
+    const errors = getErrors(reqText, reqCase);
+
+    if (!errors.length) {
+      const {
+        convertedText,
+        originalCase,
+      } = convertToCase(reqText, reqCase);
+
+      const convertResult = {
+        originalCase,
+        targetCase: reqCase,
+        originalText: reqText,
+        convertedText,
+      };
+
+      res.end(JSON.stringify(convertResult));
```

Comment:
![image](https://github.com/mate-academy/node_case-transformer-server/assets/110335429/4487ebb2-3773-407f-8b15-338473b4643a)

Same thing for line 29.